packages/jsii-pacmak/lib/markdown.ts

```diff
@@ -20,7 +20,17 @@ export function md2rst(text: string) {
   }
 
   function textOf(node: commonmark.Node) {
-    return node.literal ?? '';
+    if (node.literal) {
+      if (
+        node.prev?.literal === '\\' && // following a \ (might need escaping)
+        !/^\\.*/.exec(node.literal) && // doesn't start with a \ (not being escaped)
+        node.prev.prev?.literal !== '\\' // not two \ in a row (not already escaped)
+      ) {
+        return `\\${node.literal}`;
+      }
+      return node.literal;
+    }
+    return '';
```

Comment:
I wonder if the *correct* thing to do isn't to simply *always* escape back-slashes... If the Markdown literal has an escaped backslash, then the Python documentation must show an escaped backslash, too... Ergo we must escape even escaped escape sequences properly...

This would mean the function could basically be:

```suggestion
    return node.literal?.replace(/\\/g, '\\\\') ?? '';
```

Don't you think so?